### ISSUE TITLE
[Bug] "Load failed due to missing dependencies" on Windows

### ISSUE THREAD

**Issue by peterjaric** [sebastiankarn/Ember#4]
**Godot Version & Ember Version**
Godot: 4.2.1
Ember: Commit ef605634659813fc9fd6189d0cdc04016967035a (HEAD at the moment)

**Describe the bug**
When I start a newly cloned Ember like this `C:\[...]\Ember>Godot_v4.2.1-stable_win64.exe project.godot` on Windows, the Godot editor displays the error in the first screenshot and the error messages in the second screenshot are printed. It is possible to start the game after this, but as soon as you go into the first Shop the game crashes.

I am pretty sure I started the game in the same way on another Windows computer and then no errors were displayed and the game worked. That was not long ago and neither the game nor Godot seems to have been updated since then.

**Screenshots**
![image](https://github.com/sebastiankarn/Ember/assets/128737/8ea96e17-bb71-4eb9-8d3a-218189b892a5)

![image](https://github.com/sebastiankarn/Ember/assets/128737/e2632e5b-0688-40f6-9e02-d3645dcceb9a)

**Comment by peterjaric** [comment]
I forgot to add that I'm running Windows 11 on both computers.

**Comment by sebastiankarn** [comment]
Not sure if this fixed the problem (I never was able to reproduce the bug on my windows computer). But there was a conflict with the script in the inventory that I fixed, so hopefully this also fixed this bug!

Let us know if this solved the problem!

**Comment by peterjaric** [comment]
Yes, now it works! I just stashed the files which had been changed and then did a git pull. After that I could just start it as before. Thanks! I'll go ahead and close this issue. :)
